Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus: TEST.1
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus: TEST.2
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus: cactusjar.git
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus: exe
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus: log
Binary files /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/repos/carpet/.hg/dirstate and /home/jhar131/TESTING/einsteintoolkit/2012-07-14-020001/Cactus/repos/carpet/.hg/dirstate differ
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib/archive: __init__.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib/archive: petashare.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib/archive: uberftp.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: libutil.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: pyini.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: restartlib.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: sim-build.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: sim-info.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: sim-manage.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: sim-sync.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: sim-util.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simarchive.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simdb.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simdt.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simenv.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simlib.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simopts.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simproperties.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simremote.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simrestart.pyc
Only in /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/lib: simsubs.pyc
diff -r '--exclude=.svn' '--exclude=.git' /home/jhar131/TESTING/einsteintoolkit/2012-07-13-020001/Cactus/simfactory/mdb/optionlists/ranger-intel11.cfg /home/jhar131/TESTING/einsteintoolkit/2012-07-14-020001/Cactus/simfactory/mdb/optionlists/ranger-intel11.cfg
84,85c84,85
< BLAS_DIR  = NO_BUILD
< BLAS_LIBS = -L/opt/apps/intel/mkl/10.0.1.014/lib/em64t -mkl
---
> BLAS_DIR  = /opt/apps/intel/mkl/10.0.1.014/lib/em64t
> BLAS_LIBS = -mkl
87,88c87,88
< LAPACK_DIR  = NO_BUILD
< LAPACK_LIBS = -L/opt/apps/intel/mkl/10.0.1.014/lib/em64t -mkl
---
> LAPACK_DIR  = /opt/apps/intel/mkl/10.0.1.014/lib/em64t
> LAPACK_LIBS = -mkl
